### PR TITLE
Frontend styling pt2

### PR DESCRIPTION
**Pull request by thomasp18**
Finalized styling for the main page and updated the weather API's link

### CODE REVIEW

File: frontend/app/page.js [hunk 1]
```diff
@@ -9,89 +10,155 @@ async function getData() {
...
 }
 
+// Main component
 export default async function Home() {
   const data = await getData();
 
   return (
     <div>
-      <h1 className='display-1'>PikoWeatherer</h1>
+      <h1 className='display-1' style={{ textAlign: "center", padding: "auto", margin: "auto", marginTop: "1%" }}>PikoWeatherer</h1>
```
Comment (shubunchu) [inline frontend/app/page.js]:
```suggestion
      <h1 className='display-1 mt-1' style={{ textAlign: "center", padding: "auto", margin: "auto"}}>PikoWeatherer</h1>
```

use bootstrap-provided classes (e.g., `mt-x`, `mb-x`) for margins
https://getbootstrap.com/docs/5.3/utilities/spacing/

Comment (thomasp18) [reply frontend/app/page.js]: sir, yessir! o7

Comment (shubunchu) [reply frontend/app/page.js]: nice!

**Review by shubunchu — APPROVED** [review]

**Review by shubunchu — APPROVED** [review]
👍

**Pull request merged**